bundles/alpha.model/src/alpha/model/transformation/automation/OptimalSimplifyingReductions.xtend

```diff
@@ -88,7 +91,7 @@ class OptimalSimplifyingReductions {
 		originalSystemName = system.name
 		optimizationNum = 0
 		initialComplexity = systemBody.complexity
-		throttle = throttleLimit > 0
+		throttle = limit > 0
```

Comment:
This was a small typo that was missed during the previous simplification PR. I fixed this here.